### PR TITLE
Change dbsetup to testsetup

### PR DESCRIPTION
**Pull request by fulior**

### CODE REVIEW

File: ci_setup.sh [hunk 1]
```diff
@@ -26,8 +26,8 @@ echo "running ./adx build"
 adx build
 echo "running ./adx up"
 adx up
-echo "tunning ./adx/dbsetup"
-adx dbsetup
+echo "tunning ./adx testsetup"
+adx testsetup
```
Comment (tomeksabala) [inline ci_setup.sh:31]: adx testsetup is already run i :36
I think it should go like this:
adx build
adx up
adx testsetup
adx dc ps (it's an alias for docker-compose ps)

So removed steps:
adx dbsetup
adx restart ckan

**Review by tomeksabala — CHANGES_REQUESTED** [review]

**Comment by fulior** [comment]
test this please

File: ci_setup.sh (hunk 1 above)
Comment (fulior) [reply ci_setup.sh:31]: done

File: util/__init__.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 SUDO = os.environ.get('ADX_SUDO', '')
 
-compose = ["docker-compose"]
+compose = ["docker-compose --verbose"]
```
Comment (tomeksabala) [inline util/__init__.py]: Do we really want to enable `--verbose` for local dc stack? I'm worried that it'll create too much noise in local dev env.

**Review by tomeksabala — CHANGES_REQUESTED** [review]

File: util/__init__.py (hunk 2 above)
Comment (fulior) [reply util/__init__.py]: Removed

**Review by tomeksabala — APPROVED** [review]

**Pull request merged**